site/pmguides/src/paystack/paystack.md

```diff
@@ -0,0 +1,119 @@
+
+At the end of this guide, you'll have setup Paystack Collection and made your first test payment on Paystack.
+
+
+### Prerequisites
+1. [Sign up](https://paystack.com/signup) on Paystack and retrieve your **Test Secret API Key**. (See [guide for more](https://support.paystack.com/hc/en-us/articles/360009881600-Paystack-Test-Keys-Live-Keys-and-Webhooks))
+2. Visit the [Paystack Postman Workspace](https://www.postman.com/paystack-developers/workspace/paystack-api/overview)
+3. Open our [Developer Docs](https://paystack.com/docs) and [API Reference](https://paystack.com/docs/api/) to get more detailed explanations on the APIs we’ll cover
+
+
+### What You’ll Learn
+
++ Fork the Paystack collection on your Postman
++ Authorize Paystack requests using environment variables
++ Initialize a test transaction with Postman
++ Complete a test transaction on Paystack
+
+<!-- ------------------------ -->
+
+## Setup the collection and environment
+
+Duration: 2
+
+To get started, fork the collection from [Paystack Postman Workspace](https://www.postman.com/paystack-developers/workspace/paystack-api/overview) add it to our workspace. 
+Forking a collection makes a copy of the collection in your workspace that you can modify. Checkout [this guide from Postman](https://learning.postman.com/docs/collaborating-in-postman/using-version-control/forking-entities/) that explores the topic more.
+
+
+Click on the three dots next to the Paystack collection and select the “Create a fork” option. On the next page, you can set a label for the fork, and specify what workspace it should be created in. Set the values and click on “Fork Collection”.
+
+![Forking the Paystack collection](assets/postman_fork_collection.png)
+
+<!-- ------------------------ -->
+
+
+## Set your API key
+
+Duration: 2
+
+In your newly forked collection, go to the Authorization tab and replace the value of the Token input with your Test Secret Key from the Paystack Dashboard. You can use [environment variables](https://learning.postman.com/docs/sending-requests/variables/) on Postman to set up secret keys for different integrations. This way, you only switch the environment and your API keys change.
+
+![Set your API key](assets/postman_setup_api_key.png)
+
+<!-- ------------------------ -->
+
+
+## Make your first API call
+
+Duration: 1
+
+You're now ready to make yoru first call to [Initialize Transaction endpoint](https://paystack.com/docs/api/transaction#initialize). This is the endpoint used to create a transaction on Paystack. It returns a url where you redirect the customer to pay and complete the transaction. There are only two required parameters in here, a valid `email` and  `amount` to charge.
+
+> aside positive
```

Comment:
I'm not sure what this means. Should this be a **Note**?

Reply:
@mjgutermuth  Thanks for reviewing this!

I've made the changes as requested and also updated the copy here to make it a bit more clearer. :)